### PR TITLE
[vlpp] update

### PR DESCRIPTION
**Pull request by autoantwort**

### CODE REVIEW

**Comment by autoantwort** [comment]
It seems that the linux build is broken

File: ports/vlpp/vcpkg.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
   "name": "vlpp",
-  "version-string": "0.11.0.0",
-  "port-version": 2,
+  "version": "1.1.0.0",
   "description": "Common C++ construction, including string operation / generic container / linq / General-LR parser generator / multithreading / reflection for C++ / etc",
   "homepage": "https://github.com/vczh-libraries/Release",
+  "license": null,
+  "supports": "!(osx & arm64)",
```
Comment (Adela0814) [inline ports/vlpp/vcpkg.json:7]: I wonder why these two triplets no longer supported, I didn't see the related documents.

File: ports/vlpp/vcpkg.json [hunk 2]
```diff
@@ -1,9 +1,10 @@
 {
   "name": "vlpp",
-  "version-string": "0.11.0.0",
-  "port-version": 2,
+  "version": "1.1.0.0",
   "description": "Common C++ construction, including string operation / generic container / linq / General-LR parser generator / multithreading / reflection for C++ / etc",
   "homepage": "https://github.com/vczh-libraries/Release",
+  "license": null,
```
Comment (Adela0814) [inline ports/vlpp/vcpkg.json:6]: The license in the port homepage is `Apache License 2.0 with extra conditions`, I think you can change it to `Apache-2.0`

**Review by Adela0814 — CHANGES_REQUESTED** [review]

File: ports/vlpp/vcpkg.json (hunk 2 above)
Comment (dg0yt) [reply ports/vlpp/vcpkg.json:6]: Please don't! Either I can rely on this field, or it is worthless for all ports. `null` means: Read the license.

Comment (autoantwort) [reply ports/vlpp/vcpkg.json:6]: > Please don't change it to Apache-2.0! Either I can rely on this field (no extra restrictions), or it is worthless for all ports. null means: Read the license - this is the right thing here.

This!!

**Comment by Adela0814** [comment]
@autoantwort  Are you trying to fix the build error of Linux？
I wonder if this port has never been built successfully on `Linux`? If so, you can add it to `supports` as `osx` and `arm64`.

**Comment by Adela0814** [comment]
@autoantwort  Are you trying to fix the build error of Linux？
I wonder if this port has never been built successfully on `Linux`? If so, you can add it to `supports` as `osx` and `arm64`.

**Comment by Adela0814** [comment]
Convert this PR to draft since there is no progress for a long time.

**Comment by Adela0814** [comment]
Closing this PR since it seems that no progress is being made. Please reopen if work is still being done.

**Pull request closed without merging**